gadget-container/entrypoint.sh

```diff
@@ -178,8 +179,7 @@ if [ "$TOOLS_MODE" = "auto" ] || [ -z "$TOOLS_MODE" ] ; then
       # Use objcopy to put the btf info in an ELF file as libbpf and cilium/ebpf
       # by default check if there is an ELF file with the .BTF section at
       # /boot/vmlinux-$KERNEL.
-      objcopy --input binary --output elf64-little --rename-section .data=.BTF $KERNEL.btf /boot/vmlinux-$KERNEL
-      rm $KERNEL.btf /tmp/vmlinux.btf.tar.xz
+      objcopy --input binary --output elf64-little --rename-section .data=.BTF *.btf /boot/vmlinux-$KERNEL
```

Comment:
If we are sure there won't be more than one file with extension btf (`*.btf`) in the same folder, then LGTM.